src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -210,7 +209,8 @@ void AppendSearchQuery(IRestRequest request, string searchQuery)
                 return;
             }
 
-            request.Resource += string.Format("search?q={0}", HttpUtility.UrlEncode(searchQuery));
+            request.Resource += "search";
+            request.AddParameter("q", searchQuery, ParameterType.GetOrPost);
```

Comment:
You also removed `HttpUtility.UrlEncode` from this line. Does AddParameter does the encoding internally?